src/state/routing/slice.ts

```diff
@@ -28,7 +28,6 @@ const protocols: Protocol[] = [Protocol.V2, Protocol.V3, Protocol.MIXED]
 
 const DEFAULT_QUERY_PARAMS = {
   protocols: protocols.map((p) => p.toLowerCase()).join(','),
-  forceMixedRoutes: true,
```

Comment:
this part should be done in a separate PR

Reply:
that way we know what we can minimally revert if issues arise

Reply:
makes sense, will cut

Reply:
#4326

Reply:
I think all that PR needs to be is `forceMixedRoutes: true`?